dd-sdk-android/src/test/kotlin/com/datadog/android/bridge/internal/BridgeRumTest.kt

```diff
@@ -160,11 +191,14 @@ internal class BridgeRumTest {
         @StringForgery(regex = "GET|POST|DELETE") method: String,
         @StringForgery(regex = "http(s?)://[a-z]+\\.com/\\w+") url: String
     ) {
+        // Given
+        val updatedContext = fakeContext + (DatadogRumMonitor.TIMESTAMP to fakeTimestamp)
```

Comment:
Why do you need this TIMESTAMP in here ? As I can see you are not asserting it anywhere in this test ?

Reply:
I am asserting that the parameter to the startResource is indeed the updatedContext ( =  original context + this timestamp)